website/content/en/preview/getting-started/getting-started-with-karpenter/cloudformation.yaml

```diff
@@ -143,7 +142,7 @@ Resources:
                   "aws:ResourceTag/kubernetes.io/cluster/${ClusterName}": "owned"
                 },
                 "StringLike": {
-                  "aws:ResourceTag/karpenter.sh/provisioner-name": "*"
+                  "aws:ResourceTag/karpenter.sh/nodepool": "*"
```

Comment:
This should keep the support for both `karpenter.sh/provisioner-name` and `karpenter.sh/nodepool` in this release. It is necessary to facilitate the transition and allow karpenter to deprovision nodes created with the previous release.

Reply:
Correct. The CF that we're going to include in the preview docs isn't going to have both of these permissions. The easiest way that I was thinking the migration could occur is if a user applies a separate policy (which would be this new policy) and then drops the old policy when they are done.

Reply:
That keeps things separate rather than having to unwind all of the permissions if they are collapsed into the same policy.